dataactbroker/handlers/fileHandler.py

```diff
@@ -1,11 +1,14 @@
 import os
+import boto
```

Comment:
Is there a reason we're not using Boto3? just `boto` is version 2.x.x, it'd be best to use the latest if possible